eZ/Publish/API/Repository/RoleService.php

```diff
@@ -376,7 +405,19 @@ public function getRoleAssignmentsForUser(User $user, $inherited = false);
     public function getRoleAssignmentsForUserGroup(UserGroup $userGroup);
 
     /**
-     * Instantiates a role create class.
+     * Removes the given role assignment.
+     *
+     * i.e. unassigns a user or a user group from a role with the given limitations
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the authenticated user is not allowed to remove a role assignment
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the role assignment was not found
+     *
+     * @param \eZ\Publish\API\Repository\Values\User\RoleAssignment $roleAssignment
+     */
+    public function unassignRoleByAssignment(RoleAssignment $roleAssignment);
```

Comment:
To be consistent with the other `unassignRole*` methods - but changing this is fine of course.